draft-pauly-oblivious-doh.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Oblivious DNS Over HTTPS"
-abbrev: Oblivious DoH
+title: "Oblivious HTTPS"
+abbrev: Oblivious HTTPS
 docname: draft-pauly-oblivious-doh-latest
```

Comment:
We should probably rev the doc name too =)

Reply:
Yep, though the group where this work takes place is then less clear to me. Would we move it to HTTPBIS?

Reply:
We could, yeah. At least discuss it.